README.md

```diff
@@ -1,176 +1,100 @@
 
-  * chained comparisons like: `a < b < c`
-  * semantic comparisons using the structural operators: `<`, `>`, `<=`, `>=`, `==`, `!=`, `===`, and `!==`
-  * combinations using: `and`, `or`, and `not`
+## Key Features
 
-### Examples
+CompareChain provides convenience macros for comparisons like:
 
-```elixir
-iex> import CompareChain
-
-# Chained comparisons
-iex> compare?(1 < 2 < 3)
-true
-
-# Semantic comparisons
-iex> compare?(~D[2017-03-31] < ~D[2017-04-01], Date)
-true
-
-# Chained, semantic comparisons
-iex> compare?(~D[2017-03-31] < ~D[2017-04-01] < ~D[2017-04-02], Date)
-true
-
-# Semantic comparisons with logical operators
-iex> compare?(~T[16:00:00] <= ~T[16:00:00] and not (~T[17:00:00] <= ~T[17:00:00]), Time)
-false
-
-# More complex expressions
-iex> compare?(%{a: ~T[16:00:00]}.a <= ~T[17:00:00], Time)
-true
-```
+- chained comparisons (`a < b < c`)
+- semantic comparisons using structural operators (`<`, `>`, `<=`, `>=`, `==`, `!=`, `===`, and `!==`)
+- combinations (`and`, `or`, and `not`)
 
-## Installation
+## Background
 
-Add `compare_chain` to your list of dependencies in `mix.exs`:
-
-```elixir
-def deps do
-  [
-    {:compare_chain, "~> 0.5"}
-  ]
-end
-```
-
-Documentation can be found at <https://hexdocs.pm/compare_chain>.
-
-## Usage
-
-Once installed, you can add:
-
-```elixir
-import CompareChain
-```
-
-to your `defmodule` and you will have access to `CompareChain.compare?/1` and `CompareChain.compare?/2`.
-
-## Background and motivation
-
-`CompareChain` was originally motivated by the following situation:
-
-> You have an interval of time bounded by a two `%Date{}` structs: `start_date` and `end_date`.
-> You want to know if some third `date` falls in that interval.
-> How do you write this?
-
-In Elixir, we'd write this as follows:
+Say you have an interval of time bounded by two `%Date{}` structs, `start_date` and `end_date`, and want to know whether or not a third `date` falls within that interval. How would you write this in Elixir?
 
 ```elixir
 Date.compare(start_date, date) == :lt and
   Date.compare(date, end_date) == :lt
 ```
 
-This is verbose and therefore a little hard to read.
-It's also potentially incorrect, though not obviously so.
-What if `date` is considered "within" the interval even if it equals `start_date` or `end_date`?
-To include the bounds in our comparison, we'd instead write the expression like this:
+The above code is verbose, somewhat hard to read, and potentially incorrect (though not obviously so). What if `date` is considered "within" the interval inclusive of the `start_date` or `end_date`? To include the bounds in the comparison, you'd instead write the expression like this:
 
 ```elixir
 Date.compare(start_date, date) != :gt and
   Date.compare(date, end_date) != :gt
+
+# …or, terser but less performant:
+Date.compare(start_date, date) in [:lt, :eq]
```

Comment:
I think I see now where I misinterpreted the note in the original:

> (We could have written `Date.compare(start_date, date) in [:lt, :eq]`, but `!= :gt` is faster.)

The accurate example here should be:

```elixir
Date.compare(start_date, date) in [:lt, :eq] and
  Date.compare(date, end_date) in [:lt, :eq]
```

Maybe the comment/note should be, like… `…or, more verbose and less performant` (if I'm taking the original use of `!= :gt` being "faster"…?

Reply:
Since the performance difference is likely negligible (and the upsell here is writing less verbose code), I went with this in a0d9eaa:

```elixir
# …or, even more verbosely:
Date.compare(start_date, date) in [:lt, :eq] and
  Date.compare(date, end_date) in [:lt, :eq]
```